test/extensions/filters/network/dubbo_proxy/utility.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+#include <initializer_list>
+
+#include "common/buffer/buffer_impl.h"
+#include "common/common/byte_order.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace NetworkFilters {
+namespace DubboProxy {
+namespace {
+
+inline void addInt16(Buffer::Instance& buffer, int16_t value) {
+  value = htobe16(value);
+  buffer.add(&value, 2);
+}
+
+inline void addInt32(Buffer::Instance& buffer, int32_t value) {
+  value = htobe32(value);
+  buffer.add(&value, 4);
+}
+
+inline void addInt64(Buffer::Instance& buffer, int64_t value) {
+  value = htobe64(value);
+  buffer.add(&value, 8);
+}
+
+inline void addInt8(Buffer::Instance& buffer, int8_t value) { buffer.add(&value, 1); }
```

Comment:
not sure if you really need these three, seems buffer.add(std::string) with string literal would do same thing.

Reply:
remove it.

Reply:
?
do you still need addInt8, addRepeated, addSeq? They can be replaced by std::string, and [string literals](https://en.cppreference.com/w/cpp/string/basic_string/operator%22%22s)?